sdks/java/extensions/arrow/build.gradle

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * License); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins { id 'org.apache.beam.module' }
+applyJavaNature(automaticModuleName: 'org.apache.beam.sdk.extensions.arrow')
+
+description = "Apache Beam :: SDKs :: Java :: Extensions :: Arrow"
+
+def arrow_version = "0.15.1"
+
+dependencies {
+    compile library.java.slf4j_api
+    compile library.java.vendored_guava_26_0_jre
+    compile project(path: ":sdks:java:core", configuration: "shadow")
+    compile "org.apache.arrow:arrow-vector:0.15.1"
```

Comment:
lets make sure to upgrade the version to 3.0.0 (4.0.0 should be released pretty soon).

Reply:
Ah thanks. I actually just noted that in the jira but it's good to have it all in one place :)